### ISSUE TITLE
Sent items appearing as drafts on Exchange

### ISSUE THREAD

**Issue by bengotow** [nylas/N1#869]
(via https://app.intercom.io/a/apps/t7k2sjgy/inbox/unassigned/conversations/1554764604)

Hi! I have several messages on two Exchange servers that are moved into Drafts instead of Sent Items when I first setup the two accounts. Is there a way to move them (manually) into Sent Items? Doesn't seem possible...

**Comment by rdewolff** [comment]
Any news?

**Comment by rdewolff** [comment]
Still not working with the last version....

**Comment by matthewbassett** [comment]
+1 this one for me, pretty much everything I send is showing up as a draft. As an added bit of information, if I send an email from my mail client on my phone, it will also show up in N1 as a draft, so the issue is not only with sent items from Nylas.

**Comment by rdewolff** [comment]
This issue is taking ages to be taken care of.

**Comment by khamidou** [comment]
Hi @rdewolff! We just shipped a fix for this issue --- can you confirm it's not happening for new emails?

regards,

Karim

**Comment by rdewolff** [comment]
Thanks, seems to work better! Will test it a bit longer and deeper and get back to you if I can find any bugs.

**Comment by kashyapchhatbar** [comment]
This doesn't work in normal IMAP accounts yet...

**Comment by bartlomn** [comment]
Still experiencing this issue. Version 0.4.5-b0ec335 with gmail account.

**Comment by matthewbassett** [comment]
The issue is partially fixed I believe. However if I start a draft (Using an exchange email account) and then wait for a bit it will create a second copy of the draft. I presume this behaviour is caused when the draft is saved to the server. It is saving the item correctly but is not using that same saved draft in the N1 client. Not sure whether to post this in #993 or #1229 or here. I believe all these issues could be related to the way drafts are synced to the server.

**Comment by jstejada** [comment]
reopening as a few people are still experiencing this issue

**Comment by markbao** [comment]
Does this issue also encompass similar issues on Gmail?

The message is getting sent correctly, but then after the message gets sent, it re-appears in the thread as a draft (so if there was 1 message in the thread, and I sent another, the first message, my reply, and the draft of my reply, a total of 3, will show up) and then disappears after a few seconds.

And the message doesn't appear at the top of my Sent Mail folder (nor in my Drafts folder).

OS X El Capitan 10.11.3 
Nylas N1 Version 0.4.9-575c091 (0.4.9-575c091)

**Comment by bram96** [comment]
I have this problem with Gmail (Google Apps)

Ubuntu 14.04
Nylas N1 Version 0.4.9-575c091

**Comment by rdewolff** [comment]
Still having problems with Exchange server...

**Comment by arslankhalidch** [comment]
i am experiencing the same issue where my emails from exchange server are sent but i cannot find them anywhere. Also, when i search anything in exchange server mailbox, i get no results back.

**Comment by bruno222** [comment]
I having this problem with Gmail

Windows 7
Nylas N1 Version 0.4.37

**Comment by mosski1** [comment]
I'm having this issue now - sent emails using exchange are being shown as drafts. I reload the app on my mac, and still shows as draft, however, the recipients are receiving the emails...

I'm running mac osx 10.11.4 & Nylas 0.4.38-27d21d1

**Comment by rdewolff** [comment]
Yeah unfortunately every month I try again, uninstall everything, install the actual version, config my accounts and try. Every time the same result : the bug has **not** been fixed.

**Comment by rdewolff** [comment]
Exchange and Gmail seems to be impacted : I suspect any hosting provider has the same issue.
